func_sig_registry/utils/abi.py

```diff
@@ -139,3 +139,24 @@ def event_definition_to_text_signature(abi: Dict[str, Any]) -> str:
             for abi_input in abi.get('inputs', [])]
         ),
     )
+
+
+def retrieve_stats_from_import_results(import_results):
+        num_processed = len(import_results)
+        
+        import_results = [
```

Comment:
I'd recommend against overwriting this variable since it is destructive, making the function argument no longer accessible.  It also adds confusion to the code flow.  It took me an extra moment to figure out what was going on down on line 153 where you do `num_processed - len(import_results)` since I didn't immediately notice that `import results` had been overwritten.

As for variable naming, maybe something like `cleaned_import_results` or you could rename the function argument to `raw_import_results`